tools/setup_git_secrets.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
```

Comment:
The problem is that although the hooks are shared, the results of the `git secrets --add` commands are stored locally. Probably the installation pieces aren't necessary, but a centralized place to put all the adds would be good.